.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
Just making a top comment to tell you to remember your README to get those 2 points. :)